### ISSUE TITLE
CycleGAN Loading Check Points -- Why use initial learning rate?

### ISSUE THREAD

**Issue by SARIHUST** [aladdinpersson/Machine-Learning-Collection#119]
Hi there, you've done a great job in implementing CycleGANs from scratch, which helped me a lot.
However, I don't quite understand the way you load checkpoints in utils.py:
```
# If we don't do this then it will just have learning rate of old checkpoint
# and it will lead to many hours of debugging \:
for param_group in optimizer.param_groups:
    param_group["lr"] = lr
```
In my understanding, the main reason we set up a checkpoint is that we may want to go on training the model later and skip the earlier training epochs. So why should we use the initial learning rate instead of the learning rate at the end of the last training procedure? Can any one help me out here, thanks a lot.
